apps/omg/test/omg/ethereum_event_listener/core_test.exs

```diff
@@ -24,165 +24,137 @@ defmodule OMG.EthereumEventListener.CoreTest do
   @service_name :name
   @request_max_size 5
 
-  test "asks until root chain height provided" do
-    create_state(0, request_max_size: 100)
-    |> Core.get_events_range_for_download(%SyncGuide{sync_height: 1, root_chain_height: 10})
+  test "will respect request_max_size if its allowed!" do
+    create_state(0, request_max_size: 10)
+    |> Core.get_events_range(%SyncGuide{sync_height: 20, root_chain_height: 10})
+    |> assert_range({1, 11})
```

Comment:
If events will be fetched for blocks 1 to 11 (included) then `request_max_size` name is misleading. In that case I would expect to fetch a total of 10 blocks, and we'll be fetching 11.